core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -3661,6 +3662,13 @@ class KafkaApis(val requestChannel: RequestChannel,
     }
   }
 
+  def handleConsumerGroupDescribe(request: RequestChannel.Request): CompletableFuture[Unit] = {
```

Comment:
Should we add a unit test in KafkaApisTest to ensure that the basic plumbing works?